### DISCUSSION TITLE
Multiple servers

### DISCUSSION THREAD

**Discussion by bigboban** [owntracks/android#1569] [Ideas & New Features]
I was forced to move from Life360 to Owntracks because Life360 Home assistant integration is probably broken forever. Owntracks is fine. BUT. I need location on TWO servers. Home assistant owntracks integration is HTTP type. But i cannot add secont http server into owntracks settings. Why? It should be easy support more servers i think.

**Comment by jpmens** [comment]
> It should be easy support more servers i think.

Really? Well, we don’t.

**Comment by hellcry37** [comment]
why not for example i could use one for dawarich and one for home assistant, why is this not a good idea especially that app will report once for two servers ?

**Comment by jpmens** [comment]
This discussion is 2 1/2 years old ...
You can already do that with OwnTracks: publish to MQTT, create as many subscribers as you wish to forward locations to as many servers as you want.

**Comment by hellcry37** [comment]
you fail. good day
